Capstone/ViewControllers/CalendarViewController.m

```diff
@@ -113,6 +127,25 @@ - (void)calendar:(FSCalendar *)calendar didSelectDate:(NSDate *)date atMonthPosi
 -(void)calendar:(FSCalendar *)calendar didDeselectDate:(NSDate *)date atMonthPosition:(FSCalendarMonthPosition)monthPosition {
 }
 
+- (BOOL)isSameMonth:(NSDate*)date1 otherDay:(NSDate*)date2 {
+    NSCalendar* calendar = [NSCalendar currentCalendar];
+    unsigned unitFlags = NSCalendarUnitYear | NSCalendarUnitMonth |  NSCalendarUnitDay;
+    NSDateComponents* comp1 = [calendar components:unitFlags fromDate:date1];
+    NSDateComponents* comp2 = [calendar components:unitFlags fromDate:date2];
```

Comment:
it would be great to clean up this code, we can move some of it into the helper function